.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: node_js
+
+node_js:
+  - '6'
+
+dist: trusty
+sudo: required
+
+env:
+  global:
+  - NODE_ENV=test
+  - LABS_AUTH_SERVICE_pgdb_password=''
+
+addons:
+  postgresql: "9.5"
```

Comment:
I have to think about how we could trick this.
With a single Travis file the brute force way is to add scripts that install various Postgres versions but that's complicated. We still have the issue that we need Postgres >= 9.5.

What versions of Pg are you thinking to test against? Travis offers 9.6 and 9.6 at this moment.